src/calculateRentalCost.js

```diff
@@ -1,10 +1,16 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
```suggestion
    return totalCost -= BIG_DISCOUNT;
```
you can immediately return in each case and use `const` for totalCount variable